config/settings.py

```diff
@@ -110,6 +110,7 @@
     'drf_yasg',
     'tinymce',
     'strawberry.django',
+    'debug_toolbar',
```

Comment:
Let's only include this if needed, graphene has some issue with django-debug-toolbar, so it's best to enable this manually when needed